src/app/postulants/postulant/postulant.component.ts

```diff
@@ -1,13 +1,12 @@
-import { Component, OnInit, OnDestroy } from '@angular/core';
 import { ActivatedRoute, Router } from '@angular/router';
-
-import { Subject } from 'rxjs';
-import { takeUntil } from 'rxjs/operators';
+import { Component, OnDestroy, OnInit } from '@angular/core';
 
 import { AuthService } from '../../core/services/auth/auth.service';
+import { Postulant } from '../../core/models/postulant.model';
 import { PostulantCredentialComponent } from '../../shared/components/postulant-credential/postulant-credential.component';
 import { PostulantsService } from '../../core/services/postulants.service';
-import { Postulant } from '../../core/models/postulant.model';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
```

Comment:
Just a comment, if you see other files, imports are sorted in the following order separated in blocks:
- Angular imports
- External imports
- Internal imports
I'm not sure why you changed the order here, but I recommend that we leave it as it was. In fact, this file doesn't need any change.